.travis.yml

```diff
@@ -7,63 +10,22 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
     - $HOME/.android/build-cache
-
-android:
-  components:
-    # use the latest revision of Android SDK Tools
-    - tools
-    - platform-tools
-    - tools
-    # The BuildTools version used by your project
-    - build-tools-27.0.3
-    # The SDK version used to compile your project
-    - android-27
-    # Additional components
-    - extra-google-m2repository
-    - extra-android-m2repository
-    - extra-android-support
-
-  licenses:
-    - 'android-sdk-preview-license-.+'
-    - 'android-sdk-license-.+'
-    - 'google-gdk-license-.+'
-
-jdk:
-  - oraclejdk8
-
+    
+env:
+  global:
+    - TARGET_VERSION=31
+    - ANDROID_BUILD_TOOLS_VERSION=30.0.2
+    - ANDROID_HOME=~/android-sdk
+    
+before_install:
+  - touch $HOME/.android/repositories.cfg
+  - wget "https://dl.google.com/android/repository/commandlinetools-linux-7302050_latest.zip" -O commandlinetools.zip
```

Comment:
Let's use newer version, forgot to update it:
https://dl.google.com/android/repository/commandlinetools-linux-8092744_latest.zip